File: android/app/src/main/java/com/tuzicao/emeraldknight/ui/CheatFragment.java
package com.tuzicao.emeraldknight.ui;

import android.os.Bundle;
import android.view.LayoutInflater;
import android.view.View;
import android.view.ViewGroup;

import androidx.fragment.app.Fragment;

import com.tuzicao.emeraldknight.R;

public class CheatFragment extends Fragment {
    @Override
    public View onCreateView(LayoutInflater inflater, ViewGroup container, Bundle savedInstanceState) {
        View view = inflater.inflate(R.layout.cheat_fragment, container, false);
        return view;
    }
}

File: android/app/src/main/java/com/tuzicao/emeraldknight/ui/CheatFragment.kt
package com.tuzicao.emeraldknight.ui

import android.os.Bundle
import android.view.LayoutInflater
import android.view.View
import android.view.ViewGroup
import androidx.fragment.app.Fragment
import com.tuzicao.emeraldknight.R

class CheatFragment : Fragment() {
    override fun onCreateView(
        inflater: LayoutInflater,
        container: ViewGroup?,
        savedInstanceState: Bundle?
    ): View? {
        return inflater.inflate(R.layout.cheat_fragment, container, false)
    }
}
